ext/PDMatsSparseArraysExt/PDMatsSparseArraysExt.jl

```diff
@@ -0,0 +1,15 @@
+module PDMatsSparseArraysExt
+
+using PDMats
+using SparseArrays
+
+using PDMats.LinearAlgebra
```

Comment:
Out of curiosity, what exactly is the rationale here to not simply `using LinearAlgebra`?

Reply:
Yeah, perhaps to avoid the impression that the extension introduces yet "additional" dependencies. Sounds reasonable.

Reply:
https://github.com/JuliaStats/LogExpFunctions.jl/pull/63